docs/guide/device-and-platform-support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Device & Platform Support"
+type: guide
+layout: docs
+parent_section: guide
+order: 5
+---
+
+Support of devices and platforms depends how well the browsers support certain
```

Comment:
Suggestion: "**Device and platform support** depends **on** how well the browsers..."